search-resources-api/src/test/java/no/unit/nva/search/SearchImportCandidateAuthHandlerTest.java

```diff
@@ -0,0 +1,294 @@
+package no.unit.nva.search;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import no.unit.nva.indexing.testutils.FakeSearchResponse;
+import no.unit.nva.search.common.FakeGatewayResponse;
+import no.unit.nva.search.common.constant.Words;
+import no.unit.nva.search.common.csv.ExportCsv;
+import no.unit.nva.search.common.records.PagedSearch;
+import no.unit.nva.search.common.records.SwsResponse;
+import no.unit.nva.search.importcandidate.ImportCandidateClient;
+import no.unit.nva.testutils.HandlerRequestBuilder;
+import nva.commons.apigateway.GatewayResponse;
+import nva.commons.core.Environment;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+import static java.util.Objects.nonNull;
+import static no.unit.nva.auth.uriretriever.UriRetriever.ACCEPT;
+import static no.unit.nva.search.common.constant.Defaults.objectMapperWithEmpty;
+import static no.unit.nva.search.common.constant.Words.COMMA;
+import static no.unit.nva.search.importcandidate.ImportCandidateParameter.SEARCH_ALL;
+import static no.unit.nva.testutils.RandomDataGenerator.randomString;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static nva.commons.core.ioutils.IoUtils.stringFromResources;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsEmptyCollection.empty;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class SearchImportCandidateAuthHandlerTest {
```

Comment:
is all this new code or just git acting up?

Reply:
`SearchImportCandidateHandlerTest `& `SearchImportCandidateAuthHandlerTest `are identical

SearchImportCandidateHandler was created first, and then authorization was required, I justed copied the handler and added one line for authorization.

New task, remove old handler....